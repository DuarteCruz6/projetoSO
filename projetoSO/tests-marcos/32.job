WRITE [(q0,76)(i0,32)]
WRITE [(h8,85)(b8,63)]
WRITE [(c9,13)(u5,74)(z2,45)]
WRITE [(p3,56)(s3,75)(m2,30)(d9,43)(b5,70)]
WRITE [(o3,56)(k1,49)(a3,45)(i3,61)]
WRITE [(l9,93)(y8,62)]
WRITE [(q2,64)(e9,81)]
READ [k1]
DELETE [q0,e9,c9,y8]
WRITE [(w4,36)(t6,86)(z1,72)]
WRITE [(d3,22)(r7,56)(e0,85)(u1,98)]
WRITE [(f4,87)]
WRITE [(m3,94)(n2,44)(o5,44)(n7,54)]
READ [e0]
READ [u5]
READ [a3]
WAIT 3000
DELETE [i0,m2,f4]
WAIT 2000
WRITE [(i7,96)(g1,65)(v7,71)(s8,1)]
DELETE [p3,e0,o3,d9]
WRITE [(v9,80)]
SHOW
DELETE [d3,m3,s8]
READ [g1]
BACKUP
WRITE [(y7,25)(e9,43)]
WAIT 2000
DELETE [v9,w4,z1,i7,b5]
BACKUP
READ [h8]
WRITE [(u3,17)(a4,100)(e4,24)]
DELETE [e9,n7]
WRITE [(c5,40)(k8,62)(y3,42)(z4,58)]
WAIT 1000
WRITE [(e3,9)(i6,0)(y1,81)]
WAIT 3000
WRITE [(q3,17)(m6,47)]
READ [c5]
WAIT 1000
BACKUP
WRITE [(z8,58)(r2,63)]
DELETE [v7,s3,g1]
READ [b8]
BACKUP
WRITE [(h5,58)(a9,75)(x7,42)(j2,72)]
WAIT 2000
SHOW
WRITE [(l4,99)]
READ [r2]
WRITE [(n3,23)(b7,28)(x2,56)(a1,74)(e9,77)]
DELETE [t6,a1,z8,i3,z2]
WRITE [(x9,87)]
READ [j2]
