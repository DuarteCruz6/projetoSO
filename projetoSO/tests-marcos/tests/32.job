BACKUP
BACKUP
WRITE [(h4,84)]
READ [h4]
WRITE [(x2,8)(t8,27)(n0,85)]
WRITE [(q2,60)]
READ [q2]
WRITE [(u2,28)(w9,30)]
WRITE [(m1,68)]
WAIT 2000
WRITE [(f9,98)]
READ [m1]
WRITE [(x0,98)(m2,10)]
