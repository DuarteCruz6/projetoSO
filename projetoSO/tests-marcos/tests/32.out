[(h4,84)]
[(q2,60)]
[(m1,68)]
